doc/src/tutorials/biomechanics/biomechanics.rst

```diff
@@ -0,0 +1,799 @@
+   lC*u2(t)*cos(q3(t))*C.x - lC*u2(t)*sin(q3(t))*C.y + lD*(u2(t)*cos(q3(t)) + u4(t))/3*D.x - lD*(u2(t)*sin(q3(t))*cos(q4(t)) - u3(t)*sin(q4(t)))/3*D.y
+   >>> Do.v2pt_theory(P3, N, D)
+   lC*u2(t)*cos(q3(t))*C.x - lC*u2(t)*sin(q3(t))*C.y + lD*(u2(t)*cos(q3(t)) + u4(t))/2*D.x - lD*(u2(t)*sin(q3(t))*cos(q4(t)) - u3(t)*sin(q4(t)))/2*D.y
+   >>> P4.v2pt_theory(P3, N, D)
+   lC*u2(t)*cos(q3(t))*C.x - lC*u2(t)*sin(q3(t))*C.y + lD*(u2(t)*cos(q3(t)) + u4(t))*D.x - lD*(u2(t)*sin(q3(t))*cos(q4(t)) - u3(t)*sin(q4(t)))*D.y
+
+There are three holonomic constraint equations needed to keep the hand
+:math:`P_4` on the lever :math:`P_1`::
+
+   >>> holonomic = (P4.pos_from(O) - P1.pos_from(O)).to_matrix(N)
+
+Define inertia
+==============
+
+The inertia dyadics can be formed assuming the lever, upper arm, and lower arm
+are thin cylinders::
+
+   >>> IA = me.Inertia(me.inertia(A, mA/12*lA**2, mA/2*lA**2, mA/12*lA**2), Ao)
+   >>> IC = me.Inertia(me.inertia(C, mC/12*lC**2, mC/12*lC**2, mC/2*lC**2), Co)
+   >>> ID = me.Inertia(me.inertia(D, mD/12*lD**2, mD/12*lD**2, mD/2*lD**2), Do)
+
+   >>> lever = me.RigidBody('lever', masscenter=Ao, frame=A, mass=mA, inertia=IA)
+   >>> u_arm = me.RigidBody('upper arm', masscenter=Co, frame=C, mass=mC, inertia=IC)
+   >>> l_arm = me.RigidBody('lower arm', masscenter=Do, frame=D, mass=mD, inertia=ID)
+
+Define forces
+=============
+
+We will simulate this system in Earth's gravitational field::
+
+   >>> gravC = me.Force(u_arm, mC*g*N.z)
+   >>> gravD = me.Force(l_arm, mD*g*N.z)
+
+The lever has inertia but we will also add a linear torsional spring and damper
+to provide some more resistance for the arm to press against and pull on::
+
+   >>> lever_resistance = me.Torque(A, (-k*q1 - c*u1)*N.z)
+
+Biceps
+------
+
+We will model the biceps muscle as an actuator that extends and contracts
+between the two muscle attachment points. This muscle can extend and contract
+given an excitation specified input and we will assume that the tendon is
+rigid. The musculotendon actuator model will be made up of two components: a
+pathway on which to act and activation dynamics that define how an excitation
+input will propagate to activating the muscle. The biceps muscle will act along
+a :obj:`~sympy.physics.mechanics.pathway.LinearPathway` and will use a specific
+muscle dynamics implementation derived from [DeGroote2016]_.
+
+Start by creating the linear pathway::
+
+   >>> biceps_pathway = me.LinearPathway(Cm, Dm)
+
+You can create an activation model that is fully symbolic or create it with the
+specific tuned numerical parameters from [DeGroote2016]_ like so
+(recommended)::
+
+   >>> biceps_activation = bm.FirstOrderActivationDeGroote2016.with_default_constants('biceps')
+
+The full musculotendon actuator model is then named and constructed with a
+matching class::
+
+   >>> biceps = bm.MusculotendonDeGroote2016('biceps', biceps_pathway,
+   ...                                       activation_dynamics=biceps_activation)
```

Comment:
What happens here if you don't give it the, seemingly, optional activation dynamics?

Reply:
It's a positional argument, I just often use keyword arguments when they're not needed if there's ore than one argument for my own clarity.